config/i18n.json

```
@@ -69,6 +69,8 @@
     "ItemUniquenessExplanation_male": "You tried to move the '{{name}}' {{type}} to your {{character}} but that destination already has that item and is only allowed one.",
     "Maintenance": "Bungie.net servers are down for maintenance.",
     "MissingInventory": "Bungie.net did not return your inventory, possibly because your privacy settings prevent it. Try logging out and logging back in.",
+    "DestinyCannotPerformActionAtThisLocation": "You cannot equip items or change mods while in an activity. Try heading to orbit or a social area. This is a limitation of the Bungie.net API, not DIM.",
+    "DestinyItemUnequippable": "You cannot equip this item. If the character was last in an activity with locked equipment, try logging into the character again.",
```

```suggestion
    "DestinyItemUnequippable": "You cannot equip this item. If this character's last activity locked your equipment, try logging into the character again.",
```
is this still clear? "was last in" feels unlikely to consistently translate well, and this is also shorter